test/contracts/bridge.ts

```diff
@@ -1,66 +1,88 @@
 import { MainnetSdk } from '@dethcrypto/eth-sdk-client'
 import { expect } from 'chai'
-import { constants, ethers, Signer, Wallet } from 'ethers'
+import { constants, ethers, Signer } from 'ethers'
 
 import {
-  Dai,
   Dai__factory,
   L1DAITokenBridge__factory,
   L1DAIWormholeBridge__factory,
+  L1Escrow__factory,
   L2DAITokenBridge__factory,
   L2DAIWormholeBridge__factory,
-} from '../typechain'
-import { deployUsingFactory, getContractFactory, mintEther } from './helpers'
-import { OptimismAddresses, waitForTx } from './helpers'
-import { getAddressOfNextDeployedContract } from './pe-utils/address'
+} from '../../typechain'
+import { L2AddWormholeDomainSpell__factory } from '../../typechain/factories/L2AddWormholeDomainSpell__factory'
```

Comment:
This could be imported on the previous line, along with the other typechain imports.